modules/library/render/src/main/java/org/geotools/renderer/crs/ProjectionHandlerFinder.java

```diff
@@ -84,7 +84,7 @@ public void setWrapLimit(int wrapLimit) {
     public static ProjectionHandler getHandler(ReferencedEnvelope renderingArea, CoordinateReferenceSystem sourceCrs, boolean wrap) throws FactoryException {
         if (renderingArea.getCoordinateReferenceSystem() == null)
             return null;
-
+        
```

Comment:
Bit of a nitpick, but this file only contains whitespace changes and should not be included in the commit.